.github/workflows/deploy.yml

```diff
@@ -6,27 +6,80 @@ on:
       - 'v*'
 
 jobs:
-  deploy:
-    runs-on: ubuntu-latest
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macOS-11]
```

Comment:
The base image for the linux build doesn't matter at all. Since the actual build happens another layer down in the "manylinux" docker images, which are actually based on centos:

https://github.com/pypa/manylinux

Reply:
We don't have a choice here, binary wheels deliver to pypi must follow these standards.

Reply:
simtools itself is not a compiled module, so the story is different (much simpler, since you upload a platform-ndependet wheel to pypi).

The question how to compile the needed binary tools by simpipe is different.